kustomize/move2kube/base/config.properties

```diff
@@ -0,0 +1,6 @@
+MOVE2KUBE_URL=http://move2kube-svc.default.svc.cluster.local:8080
+BACKSTAGE_NOTIFICATIONS_URL=http://orchestrator-backstage.orchestrator/api/notifications
```

Comment:
probably `orchestrator-backstage` should be replaced with `backstage-backstage` and `orchestrator` with `backstage-system`